ctags/parsers/autoit.c

```diff
@@ -0,0 +1,121 @@
+/*
```

Comment:
Sorry, I can not check it in Geany (maybe because CTags version is more old?), I could not transfer the changes to my PR.
I checked your PR in CTags and it seems that working fine.

Reply:
Yeah it's using features we don't yet have, but there's a PR on importing that, I just have to finish reviewing it (and it's huge).

Reply:
Is it your local, not public PR for now (i.e. in your PC)? I could not find it here and in your fork of Geany's repository.

Reply:
I think @b4n means #1263, but even that is almost 2 years olde because its so big.

Reply:
> and it's huge

I don't know how to express the degree (level?) of my impression on English :))

Reply:
"Oh [expletive deleted] " :grin:

Reply:
But it can be try to use right now, isn't it?

Reply:
I tried to use [#1263](https://github.com/geany/geany/pull/1263): as I understand, [#1263](https://github.com/geany/geany/pull/1263) requires updating from Geany's upstream and from UCTags's upstream too...
I'm sorry that I can't help :(